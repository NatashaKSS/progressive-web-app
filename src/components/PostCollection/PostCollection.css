.title {
  font-size: 1.8rem;
}

.page-container {
  padding: 1rem 2rem;
}

.posts-container {
  border: solid 1px #dbface;
}
