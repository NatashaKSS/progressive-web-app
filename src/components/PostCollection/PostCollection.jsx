import React from 'react';

import styles from './PostCollection.css';

const PostCollection = ({children}) => {
  return (
    <div className={styles.pageContainer}>
      <h2 className={styles.title}>My Collection of Healthy Food</h2>
      <div className={styles.postsContainer}>
        {children}
      </div>
    </div>
  );
}

export default PostCollection;
