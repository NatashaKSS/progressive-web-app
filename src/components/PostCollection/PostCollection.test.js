import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import PostCollection from './PostCollection';

test('<PostCollection /> should render as expected', () => {
  const component = shallow(<PostCollection />);
  const tree = toJson(component);
  expect(tree).toMatchSnapshot();
});
