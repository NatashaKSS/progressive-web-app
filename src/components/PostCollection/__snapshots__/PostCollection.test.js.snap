// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PostCollection /> should render as expected 1`] = `
<div
  className="pageContainer"
>
  <h2
    className="title"
  >
    My Collection of Healthy Foods
  </h2>
  <div
    className="postsContainer"
  >
    <Switch>
      <Route
        component={[Function]}
        exact={true}
        path="/posts"
      />
      <Route
        component={[Function]}
        path="/posts/:id"
      />
    </Switch>
  </div>
</div>
`;
